service/controller/aws/v10/key/key.go

```diff
@@ -23,7 +23,7 @@ func ChartSpecs() []key.ChartSpec {
 		},
 		{
 			AppName:           "cluster-autoscaler",
-			ChannelName:       "0-1-stable",
+			ChannelName:       "0-2-stable",
```

Comment:
Can you point me to where this is coming from?

Reply:
https://github.com/giantswarm/kubernetes-cluster-autoscaler/blob/master/helm/kubernetes-cluster-autoscaler-chart/Chart.yaml#L4 given that I understood it correctly how this works. Or did I misunderstand this?

Reply:
Current way of doing things in guest cluster charts:
You can push multiple times into the same channel if the change is minor ( this should work now and not cause any problems).
If your change is bigger then you should update the `minor` like tuomas has done here.

We don't have this written down anywhere right now and it isn't intuitive IMO. This is changing with the current WIP stories, so we deal with it until they get done.